__tests__/app.test.js

```diff
@@ -33,3 +33,26 @@ describe.only("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+    it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeInstanceOf(Array);
+          body.articles.forEach((article) => {
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String)
```

Comment:
Comment count should Ideally be a number here, it's not the end of the world but otherwise you'll have to change it each time during frontend.